app/src/main/java/com/github/libretube/services/DownloadService.kt

```diff
@@ -74,7 +73,7 @@ class DownloadService : Service() {
             )
             if (downloadType in listOf(DownloadType.VIDEO, DownloadType.AUDIO_VIDEO)) {
                 videoDownloadId = downloadManagerRequest(
-                    getString(R.string.video),
+                    "[Video] $videoName",
```

Comment:
Please use translatable strings for that (which already exist), you can access them e.g. with `getString(R.string.audio)`